src/services/survey.ts

```diff
@@ -86,13 +88,24 @@ const parseSurveyQuestion = (questionResponse: any): SurveyQuestion => {
     id: questionResponse.id,
     displayOrder: questionResponse.displayOrder,
     coverImageUrl: `${questionResponse.coverImageUrl}l`,
-    displayType: questionResponse.displayType,
     text: questionResponse.text,
     pick: questionResponse.pick,
     answers: sortRecords(parsedAnswer),
+    ...parseDisplayType(questionResponse.displayType),
   };
 };
 
+const parseDisplayType = (displayType: string) => {
```

Comment:
```suggestion
const parseDisplayType = (apiDisplayType: string) => {
```

Should we rename it so that we can easily differentiate between the raw "displayType" form the API (star|heart|smiley|...) and the displayType that we intent to use within our components? :)

Reply:
Updated to `rawDisplayType` as it feels more generic to me 😄 Updated in [b076636](https://github.com/hanam1ni/react-surveyor/pull/79/commits/b076636da5d3d95c56a7ea4528ad9ad708bceb65)